_posts/2023-06-20-trino-fest-2023-recap.md

```diff
@@ -0,0 +1,122 @@
+  patterns in Apache Iceberg](https://youtu.be/GM7EvRc7_is) presented by Ryan
+  Blue from [Tabular](https://tabular.io/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Ibis:
+  Because SQL is everywhere and so is Python](https://youtu.be/JMUtPl-cMRc)
+  presented by Phillip Cloud from [Voltron Data](https://voltrondata.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>AWS Athena
+  (Trino) in the cybersecurity space](https://youtu.be/WCuJaW7zC8k) presented by
+  Anas Shakra from [Artic Wolf](https://arcticwolf.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Skip rocks
+  and files: Turbocharge Trino queries with Hudi’s multi-modal indexing
+  subsystem](https://youtu.be/IiDOmAEOXUM) presented by Nadine Farah and  Sagar
+  Sumit from [OneHouse](https://www.onehouse.ai/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Redis &
+  Trino - Real-time indexed SQL queries (new
+  connector)](https://youtu.be/JjBtZ26IHYk) presented by Allen Terleto and
+  Julien Ruaux from [Redis](https://redis.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Let it SNOW
+  for Trino](https://youtu.be/kmpO_yM8OAs) presented by Erik Anderson from
+  [Bloomberg](https://www.bloomberg.com/company/values/tech-at-bloomberg/open-source/projects/)
+  and Yu Teng from [ForePaaS](https://www.forepaas.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>DuneSQL, a
+  query engine for blockchain data](https://youtu.be/sCJncarnGdU) presented by
+  Miguel Filipe and Jonas Irgens Kylling from [Dune](https://dune.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Data Mesh
+  implementation using Hive views](https://youtu.be/ZgcVtPFkKHM) presented by
+  Alejandro Rojas from [Comcast](https://comcast.github.io/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Inspecting
+  Trino on ice](https://youtu.be/PSGuAMVc6-w) presented by Kevin Liu from
+  [Stripe](https://stripe.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Trino
+  optimization with distributed caching on Data
+  Lake](https://youtu.be/oK1A5U1WzFc) presented by Hope Wang and Beinan Wang
+  from [Alluxio](https://www.alluxio.io/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>Starburst
+  Galaxy: A romance of many architectures](https://youtu.be/K3AlAWB-Gmg)
+  presented by Benjamin Jeter from [Datto](https://www.datto.com/).
+* [<i class="fab fa-youtube" style="color:red;padding-right:0.5em;"/>FugueSQL,
+  Interoperable Python and Trino for interactive
+  workloads](https://youtu.be/aKhI1Phfn-o) presented by [Kevin
+  Kho](https://www.linkedin.com/in/kvnkho/).
+
+## Next up
+
+With this first recap we are sharing all the video recordings with you all. Next
+up are individual blog post for each session that include further useful
+material:
```

Comment:
```suggestion
This first recap is sharing all the video recordings with you all if you can't wait.
But stay tuned, because we'll also be publishing individual blog posts and recaps
for each session, and they'll include additional useful info:
```